src/import0.ml

```diff
@@ -45,6 +45,8 @@ type 'a ref = 'a Caml.ref = { mutable contents : 'a }
 (* Reshuffle [Caml] so that we choose the modules using labels when available. *)
 module Caml = struct
 
+  include Caml
+
```

Comment:
The reason I’ve not signed the DCO or removed the draft state of this PR is that I was under the impression this part wouldn’t go in.

Also please do whatever you want with the code, it isn’t mine to begin with and this PR is only here to serve as the hotfix corresponding PR for opam-alpha-repository as far as I’m concerned.

Reply:
> it isn’t mine to begin with

Well, that only makes things worse because then we'd need signoff of the original author, or something equivalent.
Taking a quick look at the diff, I see it's mostly deletions, so hardly copyrightable, so maybe it's fine.

Anyway, @bcc32, IANAL, but it's my responsibility to make you aware that the issue exists.

Reply:
FYI, I went through and reimplemented some of these fixes from scratch internally, some of them slightly differently.  There's an issue with the Ephemeron API which means we can't be compatible with both 4.12 and 5.00 at the same time, so I haven't made any changes there yet.  @kit-ty-kate, should I close this PR seeing as your branch would still exist, or would you like to keep it open?